src/user/userController.js

```diff
@@ -9,4 +9,14 @@ exports.addUser = async (req, res) => {
       console.log(error);
       res.status(500).send({ message: "Unsuccessful, please try again later" });
     }
-  }
\ No newline at end of file
+  }
+
+  exports.deleteUser = async (req, res) => {
+      try {
+          await User.deleteOne({username: req.body.username})
+          res.status(200).send({ message: `Successfully deleted ${User.username}`});
```

Comment:
I don't think "User.username" will have a value here, have you tested it? 
you need to assign line 16 to a variable first, say const user = await .....